docs/getting-started/index.md

```diff
@@ -9,7 +10,6 @@ Developers should have some prior knowledge of the [Shopify app ecosystem](https
 
 - [Ruby](https://www.ruby-lang.org) 2.5.1+ 
 - [Node.js](https://nodejs.org) 10.0.0+
-- [curl](https://curl.haxx.se)
```

Comment:
I think we need to be clear on what we're considering requirements.  Technically, to run the CLI, all you need is Ruby.  You need Node.js _only_ if you're developing a Node.js application.  We mention `ngrok` further down this list (which is needed **if** the developer uses either the `serve` or `tunnel` command, but we need `curl` to download it and `unzip` to unpack it.)

So, when we're saying requirements, requirements for what, exactly?

Reply:
In the same way, the user will need to have Rails installed if they're building a rails app - though in that case I think we do the bundle install part for them, right?